lib/resty/upstream/healthcheck.lua

```diff
@@ -597,6 +605,16 @@ function _M.spawn_checker(opts)
         return nil, "no upstream specified"
     end
 
+    local hook_down = opts.hook_down
+    if hook_down and type(hook_down) ~= "function" then
+        return nil, 'hook_down must be a function'
```

Comment:
@p0pr0ck5 Far from me the idea of being too picky about this, but it seems like those strings (L.615 too) still use single quotes 😄